### ISSUE TITLE
Unexpected Exception in WriteException

### ISSUE THREAD

**Issue by cdhunt** [spectresystems/spectre.console#145]
I was testing with divide by zero exception in Powershell. Every other test exception I've tried works as expected.

**Input Exception**

```
$e.Exception


ErrorRecord                 : Attempted to divide by zero.
WasThrownFromThrowStatement : False
TargetSite                  : System.Object Divide(Int32, Int32)
StackTrace                  :    at System.Management.Automation.IntOps.Divide(Int32 lhs, Int32 rhs)
                                 at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
                                 at System.Management.Automation.Interpreter.DynamicInstruction`3.Run(InterpretedFrame frame)
                                 at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
Message                     : Attempted to divide by zero.
Data                        : {System.Management.Automation.Interpreter.InterpretedFrameInfo}
InnerException              : System.DivideByZeroException: Attempted to divide by zero.
HelpLink                    :
Source                      : System.Management.Automation
HResult                     : -2146233087
```

**Exception from Spectre.Console**
```
ErrorRecord                 : Exception calling "WriteException" with "2" argument(s): "Could not find color or style 'T0,T1,TRet'."
WasThrownFromThrowStatement : False
TargetSite                  : Void ConvertToMethodInvocationException(System.Exception, System.Type, System.String, Int32, System.Reflection.MemberInfo)
StackTrace                  :    at System.Management.Automation.ExceptionHandlingOps.ConvertToMethodInvocationException(Exception exception, Type typeToThrow, String methodName, Int32 numArgs, MemberInfo memberInfo)
                                 at CallSite.Target(Closure , CallSite , Type , Object , ExceptionSettings )
                                 at System.Management.Automation.Interpreter.DynamicInstruction`4.Run(InterpretedFrame frame)
                                 at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
Message                     : Exception calling "WriteException" with "2" argument(s): "Could not find color or style 'T0,T1,TRet'."
Data                        : {System.Management.Automation.Interpreter.InterpretedFrameInfo}
InnerException              : System.InvalidOperationException: Could not find color or style 'T0,T1,TRet'.
                                 at Spectre.Console.Internal.StyleParser.Parse(String text) in /_/src/Spectre.Console/Internal/Text/StyleParser.cs:line 21
                                 at Spectre.Console.Internal.MarkupParser.Parse(String text, Style style) in /_/src/Spectre.Console/Internal/Text/Markup/MarkupParser.cs:line 63
                                 at Spectre.Console.Markup..ctor(String text, Style style) in /_/src/Spectre.Console/Widgets/Markup.cs:line 39
                                 at Spectre.Console.GridExtensions.<>c.<AddRow>b__3_0(String column) in /_/src/Spectre.Console/Extensions/GridExtensions.cs:line 97
                                 at System.Linq.Enumerable.SelectArrayIterator`2.ToArray()
                                 at Spectre.Console.GridExtensions.AddRow(Grid grid, String[] columns) in /_/src/Spectre.Console/Extensions/GridExtensions.cs:line 97
                                 at Spectre.Console.Internal.ExceptionFormatter.GetStackFrames(ExceptionInfo ex, ExceptionSettings settings) in /_/src/Spectre.Console/Internal/ExceptionFormatter.cs:line 89
                                 at Spectre.Console.Internal.ExceptionFormatter.GetException(ExceptionInfo info, ExceptionSettings settings) in /_/src/Spectre.Console/Internal/ExceptionFormatter.cs:line 33
                                 at Spectre.Console.Internal.ExceptionFormatter.Format(Exception exception, ExceptionSettings settings) in /_/src/Spectre.Console/Internal/ExceptionFormatter.cs:line 23
                                 at Spectre.Console.ExceptionExtensions.GetRenderable(Exception exception, ExceptionSettings settings) in /_/src/Spectre.Console/Extensions/ExceptionExtensions.cs:line 49
                                 at CallSite.Target(Closure , CallSite , Type , Object , ExceptionSettings )
HelpLink                    :
Source                      : System.Management.Automation
HResult                     : -2146233087
```

**Comment by patriksvensson** [comment]
@cdhunt Very ironic to get an exception during exception rendering 😊

I will take a look at it in a couple of hours when I'm at a computer and submit a fix.

**Comment by cdhunt** [comment]
No rush. It seemed like an outlier, so I reported it in case it shows up in some other scenarios.

**Comment by patriksvensson** [comment]
@cdhunt This problem has been fixed and should be available in v0.31, or the latest preview version on NuGet if you want to try it out before that.

Thanks for reporting this! 👍

**Comment by cdhunt** [comment]
Looks good!

![image](https://user-images.githubusercontent.com/114173/100247217-198f5600-2f08-11eb-9411-93087ca90462.png)
